### PR TITLE
Proposed sprint for Lance/LanceDB

### PR DESCRIPTION
**Pull request by changhiskhan**
Lance is a new columnar format for multi-modal AI and billion scale vector search. It's 2000x faster than parquet for random access, comes with a scalable vector index, and is used as the storage layer for petabyte scale unstructured data lakes

### CODE REVIEW

File: data/pages-content/sprints.md [hunk 1]
```diff
@@ -196,6 +196,20 @@ Please add your sprint here!
   - [sktime getting started with contributions](https://github.com/sktime/sktime/issues/1147)
   - [Roadmap 2023-2024 issue](https://github.com/sktime/sktime/issues/4691)
   - [Good first issues](https://github.com/sktime/sktime/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+ 
+### Lance - columnar format for multi-modal AI and billion-scale vector search
+
+- Number of people: ~10 (1 organizer)
+- Room: RB 105
```
Comment (rodrigogiraoserrao) [inline data/pages-content/sprints.md:203]: Please update the occupancy in the room list above.

File: data/pages-content/sprints.md [hunk 2]
```diff
@@ -196,6 +196,20 @@ Please add your sprint here!
   - [sktime getting started with contributions](https://github.com/sktime/sktime/issues/1147)
   - [Roadmap 2023-2024 issue](https://github.com/sktime/sktime/issues/4691)
   - [Good first issues](https://github.com/sktime/sktime/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+ 
+### Lance - columnar format for multi-modal AI and billion-scale vector search
```
Comment (rodrigogiraoserrao) [inline data/pages-content/sprints.md:200]: This is a _long_ heading!
The heading should be the name of the project. 😊

```suggestion
### Lance - modern columnar data format
```
```suggestion
### Lance - columnar data format for ML
```
```suggestion
### Lance - faster parquet alternative
```

**Review by rodrigogiraoserrao — CHANGES_REQUESTED** [review]
Thanks for contributing!

Just a couple of requests before merging.

**Comment by changhiskhan** [comment]
> Thanks for contributing!
> 
> Just a couple of requests before merging.

Thanks for the review comments! PR updated.

**Review by rodrigogiraoserrao — APPROVED** [review]

**Comment by rodrigogiraoserrao** [comment]
Thanks for the sprint contribution, @changhiskhan.
See you in Prague!

**Pull request merged**